### ISSUE TITLE
Fedora nightlies broken since April 22, 2020

### ISSUE THREAD

**Issue by bllngr** [FreeRDP/FreeRDP#6605]
The nightly builds for Fedora at https://ci.freerdp.com/job/freerdp-nightly-rpm-mock/ haven't been build successfully since April 22, 2020. The last available package in the repo is for Fedora 31, from April 12.

**To Reproduce**
1. Go to http://pub.freerdp.com/repositories/rpm/fedora/
2. Open the latest available Fedora version (currently F31)
3. Click through to freerdp-nightly/x86_64/
4. See the build date of the package

**Additional context**

The OpenSuse packages at http://pub.freerdp.com/repositories/rpm/opensuse/sl15.1/freerdp-nightly/ are up-to-date, but can't be installed on Fedora 33 since they depend on older `libav` libraries.

**Comment by akallabeth** [comment]
the package builder needs a major upgrade, this is scheduled but not high priority currently.
For the time being you can take the spec file (which is in the repo) and do a local build.

**Comment by srsross** [comment]
The source RPM did not build for me.

I took the source RPM from a few days ago for Fedora, namely ``freerdp-nightly-2.0-0+0~20210128033818.868~1.git83658d212.src.rpm``.  I attempted to build this on a RHEL8 box.  (The ``freerdp-nightly.spec`` file includes conditionals based on both ``fedora`` and ``rhel``, so I assumed that it would build on RHEL as wll as Fedora.)

I've attached the file [build-failed.txt](https://github.com/FreeRDP/FreeRDP/files/5906922/build-failed.txt) which shows the ``rpmbuild`` command line, the output from that command, and then the particular errors.  In my case, the type name of ``bool`` is unknown, and the  names ``UINT32_MAX`` and ``INT64_MAX`` are undeclared.

At least for the two undeclared macros, they are defined in the file ``/usr/include/stdint.h``.  However, the log file of ``CMakeFiles/CMakeError.log`` contains a line that calls out an error: "Determining if files stdint.h exist failed with the following output..."  So the build process failed to find ``stdint.h``.

Any insight into how to build the SRPM?

**Comment by akallabeth** [comment]
@srsross did you start with a clean build directory? this looks like the generated `tar.gz` contains some leftovers that break `CMake` detections

**Comment by srsross** [comment]
Hi @akallabeth,
Sorry for the delay in responding.

You asked:

> did you start with a clean build directory?

I tried the build again, starting with no `~/rpmbuild` directory.

I executed the  command:
`$ rpmbuild -ra "freerdp-nightly-2.0-0+0~20210128033818.868~1.git83658d212.src.rpm"`
which creates that directory.  During the build, I see the same sorts of build errors such as `error: 'INT64_MAX' undeclared (first use in this function); did you mean 'INT_MAX'?` as I did previously.
